src/main/java/com/rsmart/kuali/kfs/fp/businessobject/BatchDisbursementVoucherNonResidentAlienTax.java

```diff
@@ -15,24 +15,25 @@
  */
 package com.rsmart.kuali.kfs.fp.businessobject;
 
+import java.math.BigDecimal;
+
 import org.apache.commons.lang.StringUtils;
-import org.kuali.kfs.fp.businessobject.DisbursementVoucherNonResidentAlienTax;
-import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherNonresidentTax;
 import org.kuali.rice.core.web.format.BooleanFormatter;
 
 
 /**
  * Provides String setter methods for population from XML (batch)
  */
-public class BatchDisbursementVoucherNonResidentAlienTax extends DisbursementVoucherNonResidentAlienTax {
+public class BatchDisbursementVoucherNonResidentAlienTax extends DisbursementVoucherNonresidentTax {

```

Comment:
Should this rSmart class also be renamed to follow the new "Nonresident" naming convention?

Reply:
updated